lib/synx/xcodeproj_ext/project/object/pbx_group.rb

```diff
@@ -43,32 +55,24 @@ def move_entries_not_in_xcodeproj
         end
 
         def sync_path
-          self.path = basename
+          self.path = work_pathname.relative_path_from(parent.work_pathname).to_s
```

Comment:
Not with localized stuff -- a localized file.txt should actually have paths e.g. en.lproj/file.txt, de.lproj/file.txt, etc, since its parent group has display name "file.txt" but no actual (relative) path, it's just a dummy container. Using the explicit relative path this way guarantees that we always use the full relative path even if it involves extra directories.